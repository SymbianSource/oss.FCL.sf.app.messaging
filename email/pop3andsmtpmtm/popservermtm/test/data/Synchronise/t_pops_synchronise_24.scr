+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
+OK 11 14581
+OK 11 messages
1 1444
2 1316
3 1278
4 1319
5 1235
6 1235
7 1276
8 1358
9 1358
10 1444
11 1318
.
+OK 11 messages
1 yashar.997790956:20:05631:19
2 yashar.997723757:20:02036:353
3 yashar.997726929:20:19209:1071
4 yashar.997786980:20:10598:249
5 yashar.997725402:20:09569:43
6 yashar.997790583:10:09238:71
7 yashar.997790603:10:22968:34
8 yashar.997790684:10:27442:21
9 yashar.997790800:10:01145:26
10 yashar.997791127:10:14293:22
11 yashar.997785992:10:04536:10
.
-ERR Unknown Command "CAPA"
+OK TOP 10 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997791127:10:14293:22; Tue, 14 Aug 2001 12:12:07 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1021688; 14 Aug 2001 12:00 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d92509c@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 13:01:30 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG0PK>; Tue, 14 Aug 2001 13:04:55 +0100
Message-ID: <t_pops_synchronise_message014@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 14
Date: Tue, 14 Aug 2001 12:57:07 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen
.
+OK TOP 9 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997790800:10:01145:26; Tue, 14 Aug 2001 12:06:40 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ab1019238; 14 Aug 2001 11:59 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d91be5f@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 13:00:52 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG0P1>; Tue, 14 Aug 2001 13:04:06 +0100
Message-ID: <t_pops_synchronise_message012@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 12
Date: Tue, 14 Aug 2001 12:56:18 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve 
.
+OK TOP 8 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997790684:10:27442:21; Tue, 14 Aug 2001 12:04:44 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ac1019238; 14 Aug 2001 11:59 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d912b18@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 13:00:15 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG0PB>; Tue, 14 Aug 2001 13:03:39 +0100
Message-ID: <t_pops_synchronise_message011@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 11
Date: Tue, 14 Aug 2001 12:55:52 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven 
.
+OK TOP 7 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997790603:10:22968:34; Tue, 14 Aug 2001 12:03:23 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1112189; 14 Aug 2001 11:58 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d9061ec@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 12:59:23 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG039>; Tue, 14 Aug 2001 13:02:52 +0100
Message-ID: <t_pops_synchronise_message009@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 9
Date: Tue, 14 Aug 2001 12:55:04 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Nine Nine Nine Nine Nine Nine Nine Nine Nine Nine 
.
+OK TOP 6 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997790583:10:09238:71; Tue, 14 Aug 2001 12:03:03 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1019238; 14 Aug 2001 11:59 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d90cb73@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 12:59:50 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG030>; Tue, 14 Aug 2001 13:03:19 +0100
Message-ID: <t_pops_synchronise_message010@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 10
Date: Tue, 14 Aug 2001 12:55:31 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Ten Ten Ten Ten Ten Ten Ten Ten Ten Ten 
.
+OK TOP 1 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997790956:20:05631:19; Tue, 14 Aug 2001 12:09:16 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2104698; 14 Aug 2001 12:00 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d9206e1@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 13:01:11 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG0PG>; Tue, 14 Aug 2001 13:04:33 +0100
Message-ID: <t_pops_synchronise_message013@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 13
Date: Tue, 14 Aug 2001 12:56:44 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen
.
** Read cancelled
+OK disconnecting
