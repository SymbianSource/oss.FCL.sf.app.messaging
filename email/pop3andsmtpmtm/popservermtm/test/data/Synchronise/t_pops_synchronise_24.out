STAT
LIST
UIDL
CAPA
TOP 10 1
TOP 9 1
TOP 8 1
TOP 7 1
TOP 6 1
TOP 1 1
QUIT
